dataland-frontend/src/components/forms/parts/elements/derived/ProductionSiteFormElement.vue

```diff
@@ -11,7 +11,7 @@
     label="Production Site Address"
     validation="required"
   />
-  <StringBadgeListFormField
+  <NaceCodeFormField
```

Comment:
We have the option to limit Nace code options to only the lowest level. This is also the case for the "Industry" field in Lksg for example. Double check to see which case is the right one for the list of goods and service